scripts/launch/connect-to-cluster/default.nix

```diff
@@ -64,6 +65,30 @@ let
       fallbacks: 7
   '';
   configFiles = iohkPkgs.cardano-sl-config;
+  tlsConfig = pkgs.writeText "tls-config-${environment}.yaml" ''
+    ${environments.${environment}.confKey}:
+      tls:
+        ca:
+          organization: Input Output HK
+          commonName: Cardano SL Self-Signed Root CA
+          expiryDays: 3650
+
+        server:
+          organization: Input Output HK
+          commonName: Cardano SL Server Node
+          expiryDays: 365
+          altDNS:
+            - "localhost"
+            - "localhost.localdomain"
+            - "${walletExternalIPAddress}"
```

Comment:
I'm not a fan of hard-coding this to fix one use case. If we're going to support changing the tls config file, lets change altDNS to default to localhost/127.0.0.1/etc... and let them specify an override of what altDNS's they want. I also recommend we have parameters to override the org name, cn, expiry, etc... something like:

```
customTLS = {
  organization = "Your company name";
  caName =" Your Company Name Self-Signed Root CA"
  caExpiry = 3650;
  serverName = "Cardano SL Server Node";
  serverAltDNS = [
    "yourdomain.com"
    "123.123.123.123"
    "otherdomain.com"
  ];
};
```

Reply:
@disassembler, that's a good improvement to do in the overall scheme of things, I agree.

While it might not have necessarily been the best timing for this work I have nevertheless generalised the TLS configuration.